### PR TITLE
Services are now applications

### PR DESCRIPTION
**Pull request by wallyworld**
Upstream repos have changes to accommodate the rename. This change accounts for that.

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundlechanges/68/
Test FAILed.

File: changes.go [hunk 1]
```diff
@@ -14,10 +14,10 @@ import (
 // be applied in order. The bundle data is assumed to be already verified.
 func FromData(data *charm.BundleData) []Change {
 	cs := &changeset{}
-	addedServices := handleServices(cs.add, data.Services, data.Series)
+	addedServices := handleServices(cs.add, data.Applications, data.Series)
```
Comment (frankban) [inline changes.go]: addedApps? [shrug]

Comment (frankban) [inline changes.go]: ... and handleApplications?

File: changes.go [hunk 2]
```diff
@@ -181,19 +181,19 @@ func (ch *AddRelationChange) GUIArgs() []interface{} {
...
 	// are "$deploy-42:web" or just "$deploy-42".
 	Endpoint1 string
 	Endpoint2 string
 }
 
-// newAddServiceChange creates a new change for adding a service.
-func newAddServiceChange(params AddServiceParams, requires ...string) *AddServiceChange {
-	return &AddServiceChange{
+// newAddServiceChange creates a new change for adding an application.
+func newAddServiceChange(params AddApplicationParams, requires ...string) *AddApplicationChange {
```
Comment (frankban) [inline changes.go]: newAddApplicationChange

File: changes.go [hunk 3]
```diff
@@ -297,22 +297,22 @@ func newExposeChange(params ExposeParams, requires ...string) *ExposeChange {
...
-	return []interface{}{ch.Params.Service}
+	return []interface{}{ch.Params.Application}
 }
 
-// ExposeParams holds parameters for exposing a service.
+// ExposeParams holds parameters for exposing an application.
 type ExposeParams struct {
-	// Service holds the placeholder name of the service that must be exposed.
-	Service string
+	// application holds the placeholder name of the application that must be exposed.
```
Comment (frankban) [inline changes.go]: s/application/Application/

File: changes.go [hunk 4]
```diff
@@ -270,23 +270,23 @@ type AddUnitChange struct {
...
 	}
 	return args
 }
 
-// AddUnitParams holds parameters for adding a service unit.
+// AddUnitParams holds parameters for adding an application unit.
 type AddUnitParams struct {
-	// Service holds the service placeholder name for which a unit is added.
-	Service string
+	// application holds the application placeholder name for which a unit is added.
```
Comment (frankban) [inline changes.go]: s/application holds/Application holds/

**Comment by wallyworld** [comment]
All issues fixed and new PR created https://github.com/juju/bundlechanges/pull/24
Cresting branch off this repo rather than my fork simplifies the development of other dependent repos pending getting this one landed

**Pull request closed without merging**